src/runtime_src/core/common/info_platform.cpp

```diff
@@ -103,6 +103,50 @@ add_mig_info(const xrt_core::device* device, ptree_type& pt)
   }
 }
 
+static std::map<std::string, long long> 
+p2p_convert_config(xrt_core::query::p2p_config::result_type& config)
```

Comment:
this function has been copied over from somewhere else? If so, it is better to include that file
if not then I think the correct place for this is in qr::p2p_config as a "to_string" function or something similar.